scripts/generate-google3-docs/resource-reference/generated/resource-docs/secretmanager/secretmanagersecret.md

```diff
@@ -555,7 +555,9 @@ conditions:
 externalRef: string
 name: string
 observedGeneration: integer
-observedState: {}
+observedState:
+  versionAliases:
+    string: string
```

Comment:
`status.versionAliases` is a new field I added in this PR. otherwise, it is very unclear whether the versionAliases succeeds or not. 
The `spec.versionAliases` are already there.

Reply:
Yeah, I was confused by github's presentation of the diff.  map[string]string makes sense.  It would be nice if it was map[string]int64 in KRM and proto, but it may encounter some bugs with big values.